### DISCUSSION TITLE
Disabling web login in Airflow 2.0.1

### DISCUSSION THREAD

**Discussion by shjasti** [apache/airflow#14548] [Q&A] [answered]
Airflow Version: 2.0.1
Python 3.7

Is there a way to completely deactivate web login in Airflow 2.0.1. There were a few questions on stack overflow regarding this, but unfortunately they went unanswered. I have set `authenticate = False` and `rbac = False` in my airflow.cfg file but I was wondering if there was something I needed to add in webserver_config.py to completely disable login, if that's at all possible?

**Comment by junnplus** [comment, marked as answer]
`Disabling web login` means that the user can’t log in？or user does not need to log in?

**Reply by mdmischief** [reply]
Eliminate the need to enter a user/password to log into the system.

**Reply by junnplus** [reply]
@mdmischief see https://airflow.apache.org/docs/apache-airflow/2.0.1/security/webserver.html?highlight=webserver#id4
You can try set `AUTH_ROLE_PUBLIC = 'Admin'` in webserver_config.py
